### PR TITLE
Seo stuff

### PR DESCRIPTION
**Pull request by Sijibomii**
In this pull request :
I deployed to netlify link : https://friendly-euler-983183.netlify.app/
I added social preview
I tested site for google mobile test. 
I added favicon
I test site on web dev audit and site checker pro.
On web dev audit and site checker pro, I implemented the changes I was asked to implement but the score does not seem to change despite all my changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,10 +2,216 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta property="og:title" name="social media title" itemprop="name" content="Resume for Mcdonald a frontend web developer with experience as a visual and brand designer based in lagos Nigeria.">
```
Comment (bahdcoder) [inline index.html:5]: SEO and responsiveness looking great! Nice work !

**Review by bahdcoder — APPROVED** [review]
Overall great job. Everything works fine . Very responsive on various screens, and very accurate to the initial design. 

One thing I'm worried about is your git workflow . Can you explain it to me? Each pull request is supposed to have only the file changes for the specific task you did, but that doesn't seem to be the case .

**Comment by Sijibomii** [comment]
> Overall great job. Everything works fine . Very responsive on various screens, and very accurate to the initial design.
> 
> One thing I'm worried about is your git workflow . Can you explain it to me? Each pull request is supposed to have only the file changes for the specific task you did, but that doesn't seem to be the case .

I don't have any git workflow sir. I'm just using the pull request functionality for the first time. I tried to follow the steps you showed us in the assignment video so I'm not sure but I might be missing a step.

**Comment by Sijibomii** [comment]
Also sir, Can I merge the pull request my self or I should wait for you?

**Pull request merged**